package.json

```diff
@@ -9,5 +9,9 @@
         "service": "cd service && yarn run dev",
         "client": "cd client && yarn run dev"
     },
-    "packageManager": "yarn@1.22.19"
+    "packageManager": "yarn@1.22.19",
+    "dependencies": {
```

Comment:
update to package.json should be done in client one as not all modules depend on this.